build/debian-iptables/Makefile

```diff
@@ -16,12 +16,12 @@
 
 REGISTRY?="staging-k8s.gcr.io"
 IMAGE=$(REGISTRY)/debian-iptables
-TAG?=v10.2
+TAG?=v11.0
```

Comment:
why not `v10.3` ?

Reply:
I generally reserve the .N bumps for rebases of the existing image. We actually removed a package here.